staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller.go

```diff
@@ -316,6 +316,7 @@ func (c *celAdmissionController) Validate(
 				versionedAttr = va
 			}
 
+			// TODO this is where the match code should go
 			decisions := bindingInfo.validator.Validate(versionedAttr, param)
```

Comment:
I agree, I think this is the right place.

Reply:
By right place, I mean inside the validate call, like you have below.

Reply:
ack, thanks!